### DISCUSSION TITLE
How to depend on local git repos?

### DISCUSSION THREAD

**Discussion by saikyun** [janet-lang/janet#586] [Q&A] [answered]
Is it possible to depend on local git repos in project.janet?

If not, what is the alternative, running `jpm install` in the local repo every time it is changed?

**Comment by pepe** [comment]
I am not sure, but have you tried to file:// at the beginning of the repo URL?

**Comment by pyrmont** [comment, marked as answer]
As @pepe suggested, you can do this using `file://`.

Note that you need to provide the absolute path from root to the directory containing the `.git` directory (so that you'll have `file://` followed by a `/`). For example:

```
:dependencies ["file:///absolute/path/to/repo"])
```

**Reply by saikyun** [reply]
Sorry for the late reply, but thank you! I tried file but with only // :’D
